### ISSUE TITLE
Sample call to create a new Subscription does not work without __metadata.type

### ISSUE THREAD

**Issue by ejschoen** [SharePoint/sp-dev-docs#7329]
Creating a new subscription by posting to the /subscriptions endpoint under a list appears to require the body of the request contain "__metadata": {"type":"Microsoft.SharePoint.Webhooks.Subscription"} in addition to values for resource, notificationUrl, and expirationDateTime.  Otherwise, SharePoint returns:

"{\"error\":{\"code\":\"-1, Microsoft.SharePoint.Client.InvalidClientQueryException\",\"message\":{\"lang\":\"en-US\",\"value\":\"
An entry without a type name was found, but no expected type was specified. 
To allow entries without type information, the expected type must also be specified when the model is specified.\"}}}"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dc58af9c-d9b8-5ed0-6117-2db7a1d425de
* Version Independent ID: 7cc8696c-5946-0414-b5be-c7561d0e31b5
* Content: [Create a new subscription](https://docs.microsoft.com/en-us/sharepoint/dev/apis/webhooks/lists/create-subscription)
* Content Source: [docs/apis/webhooks/lists/create-subscription.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/apis/webhooks/lists/create-subscription.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by bcameron1231** [comment]
Hi, are you passing **application/json;odata=verbose**? 

I *think* if you are passing a verbose header, you need to specify the __metadata attribute.

**Comment by ejschoen** [comment]
You are correct that I am passing ;odata=verbose.  I've got general purpose method for accessing SharePoint that deals with token authentication and takes a parameter for the HTTP method verb plus additional parameters as needed.  But it always passes Content-Type:application/json;odata=verbose in the headers.
